### ISSUE TITLE
Unexpected error occurred: stopping trading

### ISSUE THREAD

**Issue by jwest31337** [tyler-pruitt/robinhood_crypto_trader#2]
Hello again Mr. Tyler,

I gave your other code repo a shot here, but I hit another problem. Here's what I'm getting when I start engaging in a live trading process, using the example code you gave:

configuration test: PASSED
Enter Robinhood code for validation: 003852
login successful
cryptos:  ['DOGE', 'MATIC', 'SHIB', 'XLM']
======================ITERATION 1======================
mode: live
runtime: 9.0 seconds
equity: $53.2
crypto holdings:
	0 DOGE at $0.092247
	0 MATIC at $1.022241
	0 SHIB at $1.248e-05
	0 XLM at $0.09284
crypto equity: $0.0
cash: $53.2
crypto equity and cash: $53.2
profit: +$0.0 (+0.0%)
Found Additional pages.
Loading page 2 ...
Loading page 3 ...
Loading page 4 ...
Loading page 5 ...
Loading page 6 ...
Loading page 7 ...
Loading page 8 ...
Loading page 9 ...
Loading page 10 ...
Loading page 11 ...
number of pending orders: 0
number of orders: {'DOGE': 0, 'MATIC': 0, 'SHIB': 0, 'XLM': 0}

DOGE = $0.092214
An unexpected error occured: stopping trading
logout successful
Error message: (<class 'NameError'>, NameError("name 'period' is not defined"), <traceback object at 0x7fe7f03bde00>)
already logged out: logout() can only be called when currently logged in

Did I enter something incorrectly in the config? Thank you again for all your help. I'll continue testing anything you release. Cheers!

**Comment by tyler-pruitt** [comment]
Hey again jwest31338,

Thank you very much for trying this repository and reporting your issue. 

TL;DR
Which version of the code/repository did you use? Did you download the code directly from the repository or from the official release on PyPI? If you can't find out which version you used, do you know when it was last modified? You can look to see if the top of trader.py says anything about these two things. Did you use the `boll` function for determining the trade? Also, did you pass any arguments into it through `builtin_trade_function_arguments`? If so, which arguments did you use?

I think it might be due to changes I made after v1.0.7 in this repository. The last official Python package version is 1.0.7 and it is the latest release on PyPI as well. Which version are you using? If you installed the package on PyPI from pip, then you should have version 1.0.7 and it should be running smoothly as far as I know. If you used the latest code from this repository and downloaded and installed it directly, then you probably have the pre-production code for version 1.0.8. You can also check the top of the trader.py file to see if it pre-production version 1.0.8. If it doesn't say, then it is probably v1.0.7. If you can't determine the version, then the time it was last modified would be helpful for me to know if you can find it at the top of the trader.py file.

I believe that your config should be correct. Although, it may come any arguments that you passed into `builtin_trade_function_arguments` if you used `boll` for your trade function. This may be because the Trader class' `check_config` method does not check to make sure the arguments to `builtin_trade_function_arguments` are correct since they can vary to a large degree for different functions used to determine the trade. If this is the case, I can add specific tests on the arguments passed through `builtin_trade_function_arguments` for each trading function. Did you use the `boll` function for determining the trade? Also, did you pass any arguments into it through `builtin_trade_function_arguments`? If so, which arguments did you use?

Thanks again for your reporting and testing!
